KsApi/models/ErroredBackingsEnvelopeTests.swift

```diff
@@ -4,173 +4,13 @@ import ReactiveSwift
-                  "finalCollectionDate": nil,
-                  "fxRate": 1.0,
-                  "friends": [
-                    "__typename": "ProjectBackerFriendsConnection",
-                    "nodes": []
-                  ],
-                  "goal": [
-                    "__typename": "Money",
-                    "amount": "19974.0",
-                    "currency": "USD",
-                    "symbol": "$"
-                  ],
-                  "image": [
-                    "__typename": "Photo",
-                    "id": "UGhvdG8tMTEyNTczMzY=",
-                    "url": "https://ksr-qa-ugc.imgix.net/assets/011/257/336/a371c892fb6e936dc1824774bea14a1b_original.jpg?ixlib=rb-4.0.2&crop=faces&w=1024&h=576&fit=crop&v=1463673674&auto=format&frame=1&q=92&s=87715e16f6e9b5a26afa42ea54a33fcc"
-                  ],
-                  "isProjectWeLove": false,
-                  "isWatched": false,
-                  "launchedAt": 1_620_662_504,
-                  "location": [
-                    "__typename": "Location",
-                    "country": "NG",
-                    "countryName": "Nigeria",
-                    "displayableName": "Nigeria",
-                    "id": "TG9jYXRpb24tMjM0MjQ5MDg=",
-                    "name": "Nigeria"
-                  ],
-                  "name": "Mouth Trumpet Robot Cats",
-                  "pid": 1_234_627_104,
-                  "pledged": [
-                    "__typename": "Money",
-                    "amount": "65.0",
-                    "currency": "USD",
-                    "symbol": "$"
-                  ],
-                  "slug": "afeestest/mouth-trumpet-robot-cats",
-                  "state": "LIVE",
-                  "stateChangedAt": 1_620_662_506,
-                  "url": "https://staging.kickstarter.com/projects/afeestest/mouth-trumpet-robot-cats",
-                  "usdExchangeRate": 1.0
-                ],
-                "reward": nil,
-                "sequence": 5,
-                "shippingAmount": [
-                  "__typename": "Money",
-                  "amount": "0.0",
-                  "currency": "USD",
-                  "symbol": "$"
-                ],
-                "status": "errored"
-              ]],
-              "totalCount": 1
-            ],
-            "id": "VXNlci0xNDcwOTUyNTQ1",
-            "imageUrl": "https://ksr-qa-ugc.imgix.net/missing_user_avatar.png?ixlib=rb-4.0.2&blur=false&w=1024&h=1024&fit=crop&v=&auto=format&frame=1&q=92&s=e17a7b6f853aa6320cfe67ee783eb3d8",
-            "name": "Hari Singh",
-            "uid": "1470952545"
-          ]
-        ]
-      ]
+  func testErroredBackingsEnvelope_Success() {
+    let producer = ErroredBackingsEnvelope.producer(from: FetchUserBackingsQueryTemplate.valid.data)
 
-    let data = GraphAPI.FetchUserBackingsQuery.Data(unsafeResultMap: unsafeResultMap)
+    /** TODO: Ensure `ErroredBackingsEnvelope` conforms to `Decodable` before testing
```

Comment:
Can you explain what's going on here? Wouldn't the envelope conform to Decodable without any issue?

Reply:
[explanation](https://github.com/kickstarter/ios-oss/pull/1589#discussion_r693148967)